src/web_app/chrome_app/manifest.json

```diff
@@ -5,7 +5,7 @@
   "manifest_version": 2,
   "app": {
     "background": {
-      "scripts": ["js/background.js"]
+      "scripts": ["js/apprtc.debug.js", "js/background.js"]
```

Comment:
I'll add an issue to do this. Probably worth it to get things set up correctly sooner vs later.